.prettierignore

```diff
@@ -0,0 +1,9 @@
+node_modules
+artifacts
+cache
+coverage*
+gasReporterOutput.json
+dist
+typechain
+CHANGELOG.md
```

Comment:
Can remove if the changelog plugin is removed from releaserc

Reply:
[09021fc](https://github.com/VenusProtocol/token-bridge/pull/1/commits/09021fc9288402f1e1a1f75db0f4c001860f4ad5)